mxcube3/ui/components/SampleView/SampleControls.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
```

Comment:
Personally I have nothing against ternary operator.

But we decided to comply with some linting rules, so... I propose to either replace it with the more
verbose version:

' ' '
if (clickCentring) {
  sendAbortCentring()
} else {
  sendStartClickCentring()
}
'''

Or to globally change the rules to add something like [no-unneeded-ternary](http://eslint.org/docs/rules/no-unneeded-ternary), if it works in this case.